### DISCUSSION TITLE
Grouping tabs in opening a new tab from the current one

### DISCUSSION THREAD

**Discussion by hayata-yamamoto** [okaryo/TabTabTab#383] [Ideas]
Hi, Could you have room to support a new grouping rule of this feature? If possible, I'd say, I suggest a grouping feature by tab opening action.

I usually use this feature to sort and file corresponding to my brain behavior. For example, when I review others' PRs, I usually focus on GitHub domain. After finished my work, I can discard code review context by deleting github tabs for all. 

However, sometimes I face difficult situation by grouping by domain. For example, I usually work with GitHub and Notion to complete my work. GitHub manages my codebases. Notion shows design, documentations and tasks. When I should review a single PR, I must read and check with Notion and GitHub. 

Usually, I probably read notion pages or github Issues, PRs or something. In this situation, the current grouping feature is bit annoying for me because grouping feature automatically allocate tabs according to domain even if it's opened from GitHub PRs.

So, if this new feature support, I probably collect tabs according to my affairs. what do you think of this?

**Comment by okaryo** [comment]
Hi,

Thanks for the suggestion! I believe this can already be achieved with the "Apply Auto-Grouping to Current Tab Only" setting, which you can find in the auto-grouping settings. If you turn it on, new tabs opened from the current tab won’t be included in the auto-grouping. Let me know if that helps!

<img width="383" alt="Screenshot 2024-09-13 at 0 15 00" src="https://github.com/user-attachments/assets/a049bdbf-4f9c-4f35-9326-e66917e8b99c">

**Reply by hayata-yamamoto** [reply]
Great, thanks. I try it.
